xprocspec-runner/pom.xml

```diff
@@ -24,7 +24,7 @@
     <dependency>
       <groupId>org.daisy.xprocspec</groupId>
       <artifactId>xprocspec</artifactId>
-      <version>[1.0.0,2.0.0)</version>
+      <version>1.1.1-SNAPSHOT</version>
```

Comment:
The next version of xprocspec will probably be 1.2.0 and not 1.1.1. I'm working on it but have some issues left that I want to fix before releasing it.